node-genai/README.md

```diff
@@ -0,0 +1,42 @@
+# Node.js GenAI Application
+
+A Node.js-powered GenAI app you can run locally using your favorite LLM — just follow the guide to get started.
+
+## Environment Variables
+
+The application uses the following environment variables:
+
+- `LLM_BASE_URL`: The base URL of the LLM API
+- `LLM_MODEL_NAME`: The model name to use
```

Comment:
This needs to be changed in all apps to use `LLM_MODEL`, since that's the URL Compose injects.